piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        void startGame(StartCallback callback);
+
+        RoundViewModel onNotesPlayed(CompletionCallback callback, Note... note);
```

Comment:
should the model know the viewmodel? In my understanding of MVP, M and V do not meet.

It would be the presenter or a layer between the presenter and the View that creates the ViewModel.